inetum-realjobs-backend/src/main/java/world/inetum/realdolmen/realjobs/controllers/AuthenticationController.java

```diff
@@ -67,4 +68,21 @@ public MessageResponse signUp(@Valid @RequestBody SignupRequest signUpRequest) {
 
         return new MessageResponse("User registered successfully!");
     }
+
+    @PostMapping("/forgotPassword")
+    @ResponseStatus(HttpStatus.NO_CONTENT)
+    public void forgotPassword(@Valid @RequestBody ForgotRequest forgotRequest) {
```

Comment:
Is a dto really necessary here?

Reply:
Yes, the only other way would be to directly send the email as the body, but that's a fairly bad practice. Sending it as JSON (and thus using a dto) is the best imo.